### PR TITLE
Feature 65

### PR DESCRIPTION
**Pull request by MrPekar98**
This PR is linked to issue #65.
https://github.com/aau-giraf/weekplanner/issues/65

### CODE REVIEW

File: lib/models/settings_model.dart [hunk 1]
```diff
@@ -72,6 +75,9 @@ class SettingsModel implements Model {
   /// Flag for indicating whether or not greyscale is enabled
   bool greyscale;
 
+  /// defines how a completed activity is represented
```
Comment (Seraph2) [inline lib/models/settings_model.dart:78]: Capital D :)

File: lib/models/settings_model.dart [hunk 2]
```diff
@@ -72,6 +75,9 @@ class SettingsModel implements Model {
...
+  int completedActivityOption;
```
Comment (klogeat) [inline lib/models/settings_model.dart:79]: Maybe this should be an enum similar to the defaultTimer.

File: test/models/settings_model_test.dart [hunk 3]
```diff
@@ -17,6 +17,7 @@ void main() {
     'theme': 1,
     'nrOfDaysToDisplay': 7,
     'greyScale': false,
+    'completedActivityOption': 2,
```
Comment (klogeat) [inline test/models/settings_model_test.dart:20]: This should probably also be checked in some expect statement in the tests. Similar to how it is done on lines 35-53.

**Comment by antonhpalmer** [comment]
There already exists an enum `CompleteMark`:
```
enum CompleteMark{
  Removed, Checkmark, MovedRight
}
```
Why not use this? Or is it meant for something else?

**Comment by MrPekar98** [comment]
No changes required.

**Pull request closed without merging**